website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,297 @@
+from events.models.event import Event
+from payments.models import Payment
+from members.models import Member
+from moneybirdsynchronization.models import Contact
+from moneybirdsynchronization.administration import HttpsAdministration, Administration
+from moneybirdsynchronization import emails
+from thaliawebsite import settings
+from django.template.defaultfilters import date
+
+import datetime
+
+class MoneybirdAPIService():
+
+    def __init__(self):
+        api = HttpsAdministration(settings.MONEYBIRD_API_KEY, settings.MONEYBIRD_ADMINISTRATION_ID)
```

Comment:
Maybe even better, don't hardcode `settings.MONEYBIRD_API_KEY, settings.MONEYBIRD_ADMINISTRATION_ID` in the constructor but make it arguments. This uncouples your moneybird code from concrexit